subsys/bluetooth/audio/csip_crypto.h

```diff
@@ -20,13 +20,14 @@
  * used in RSIs - Used by the Coordinated Set Identification service and
  * profile.
  *
- * @param sirk The 16-byte SIRK
- * @param r 3 byte random value
- * @param out The 3 byte output buffer
+ * @param sirk  16 byte LS byte first SIRK
+ * @param r     3 byte LS byte first random value
+ * @param out   3 byte LS byte first output buffer
```

Comment:
The reason I wrote it as above is because this is how its been done previously, in e.g. `zephyr/include/zephyr/bluetooth/crypto.h`.
I don't have a particular preference on this, but I do look consistency :D
You still think I should change?